src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -978,55 +982,10 @@ export class ResultsViewPageStore {
     });
 
 
-    readonly filteredAlterationsDataByUnflattenedOQLLine = remoteData<
-    IQueriedMergedTrackCaseData[]
->({
-    await: () => [
-        this.filteredAlterations,
-        this.annotatedMolecularData,
-        this.selectedMolecularProfiles,
-        this.defaultOQLQuery,
-        this.samples,
-        this.patients
-    ],
-    invoke: () => {
-        const data = [...(this.filteredAlterations.result!), ...(this.annotatedMolecularData.result!)];
-        const accessorsInstance = new accessors(this.selectedMolecularProfiles.result!);
-        const defaultOQLQuery = this.defaultOQLQuery.result!;
-        const samples = this.samples.result!;
-        const patients = this.patients.result!;
-
-        if (this.rvQuery.oqlQuery.trim() === '') {
-            return Promise.resolve([]);
-        } else {
-            const filteredAlterationsByOQLLine: UnflattenedOQLLineFilterOutput<AnnotatedExtendedAlteration>[] = (
-                filterCBioPortalWebServiceDataByUnflattenedOQLLine(
-                    this.rvQuery.oqlQuery,
-                    data,
-                    accessorsInstance,
-                    defaultOQLQuery
-                )
-            );
-
-            return Promise.resolve(filteredAlterationsByOQLLine.map(
-                (oql) => ({
-                    cases: groupDataByCase(oql, samples, patients),
-                    oql,
-                    mergedTrackOqlList: filterSubQueryData(
-                        oql, defaultOQLQuery,
-                        data, accessorsInstance,
-                        samples, patients
-                    )
-                })
-            ));
-        }
-    }
-});
-
     readonly isSampleAlteredMap = remoteData({
-        await: () => [this.filteredAlterationsDataByUnflattenedOQLLine, this.samples],
+        await: () => [this.putativeDriverFilteredCaseAggregatedDataByUnflattenedOQLLine, this.samples],
```

Comment:
i wonder if this is getting TOO descriptive?   do we ever have this data NOT putative driver filtered?

Reply:
good point.. with this pull request, all our data is now putative driver filtered. ill remove that from the names.